src/mcts/search.cc

```diff
@@ -318,8 +318,43 @@ void Search::MaybeTriggerStop() {
   // Don't stop when the root node is not yet expanded.
   if (total_playouts_ == 0) return;
 
+  bool kldgain_too_small = false;
+  if (params_.GetMinimumKLDGainPerNode() > 0 &&
+      total_playouts_ + initial_visits_ >
+          prev_dist_visits_total_ + params_.GetKLDGainAverageInterval()) {
```

Comment:
Probably doesn't matter much, but this `>` (instead of `>=`) will be off by 1 if you're expecting it to check at 200, 300, 400, etc. Right now this checks at 202, 303, 404, etc.

Reply:
As you say it doesn't matter much - since we only update per batch and our batch size is rarely 1.  But fixed all the same.